templates/common/infra/bicep/core/host/functions.bicep

```diff
@@ -7,6 +7,7 @@ param applicationInsightsName string = ''
 param appServicePlanId string
 param appSettings object = {}
 param clientAffinityEnabled bool = false
+param enableOryxBuild bool = false
```

Comment:
Curious... Did we need to set `enableOryxBuild` explicitly? Or is there a different reason to surface this?

Reply:
Yes Functions teams said enableOryxBuild needs to be explicitly set to False for the out of process model.